internal/provider/user_role_grant_resource_test.go

```diff
@@ -445,10 +439,10 @@ func TestParseRoleGrantResourceID(t *testing.T) {
 	}
 }
 
-func getTestRoleGrantResourceConfig(clusterName string, skipOrgTest bool) string {
+func getTestRoleGrantResourceConfig(clusterName string, removeOrgResource bool) string {
```

Comment:
just changed the variable name to be more descriptive/accurate

Reply:
how about `excludeOrgResource`?  using remove here makes it sound like we're actively removing something.

Reply:
or you could flip it and use `includeOrgResource`.  Sometimes its easier for people to think in terms of positives instead of negatives.  But up to you.

Reply:
switched to `includeOrgResource` instead